CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(imu_vn_100)
 
-add_definitions("-std=c++11")
+set(CMAKE_CXX_FLAGS "-std=gnu++11")
```

Comment:
Running
    catkin_make --pkg imu_vn_100 --cmake-args -DCMAKE_BUILD_TYPE=Release

Results in this error on OS X:
![image](https://user-images.githubusercontent.com/1617568/28843233-0ca40fe2-76c6-11e7-8fd5-417a906662c1.png)